apps/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) BBP/EPFL 2011-2015
+#                        Grigori.Chevtchenko@epfl.ch
+# All rights reserved. Do not distribute without further notice.
+
+include_directories(${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_BINARY_DIR})
+
+set(RESTCONNECTOR_SOURCES RESTConnector.cpp)
+set(RESTCONNECTOR_LINK_LIBRARIES RESTConnectorLib)
```

Comment:
Lib in RESTConnectorLib is redundant, as the resulting file is named libRESTConnectorLib.so